src/pages/RepoPage/CoverageTab/subroute/Fileviewer/Fileviewer.jsx

```diff
@@ -1,12 +1,23 @@
 import { useParams } from 'react-router-dom'
 
+import { TierNames, useTier } from 'services/tier'
+import { useFlags } from 'shared/featureFlags'
 import RawFileviewer from 'shared/RawFileviewer'
 import { useTreePaths } from 'shared/treePaths'
 import Breadcrumb from 'ui/Breadcrumb'
 
 function FileView() {
   const { treePaths } = useTreePaths()
-  const { ref: commit } = useParams()
+  const { provider, owner, ref: commit } = useParams()
+
+  const { coverageTabFlagMutliSelect } = useFlags({
+    coverageTabFlagMutliSelect: false,
```

Comment:
Shouldnt we also gate this by the `multipleTier` flag? Cause if multiselect flags is released first, then this would be shown to the public